docs/sources/flow/reference/components/prometheus.integration.process_exporter.md

```diff
@@ -0,0 +1,92 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.​process_exporter
+---
+
+# prometheus.integration.process_exporter
+The `prometheus.integration.process_exporter` component embeds
+[process_exporter](https://github.com/ncabatoff/process-exporter) for collecting process stats mined from `/proc`.
+
+## Usage
+
+```river
+prometheus.integration.process_exporter "LABEL"{
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`procfs_path`       | `string`                 | procfs mountpoint. | `/proc` | no
+`track_children`    | `bool`                   | If a proc is tracked, also track any of it's children that are not part of their own group. | `true` | no
+`track_threads`     | `bool`                   | Report on per-threadname metrics as well. | `true` | no
+`gather_smaps`      | `bool`                   | Gather metrics from the smaps file, which contains proportional resident memory size. | `true` | no
+`recheck_on_scrape` | `bool`                   | Recheck process names on each scrape. | `true` | no
+`process_names`     | `process_matcher_config` | A collection of matching rules to use for deciding which processes to monitor. Each config can match multiple processes, which will be tracked as a single process "group."
+
+
+### `process_matcher_config`
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`name`       | `string`                         | <pre><p>The name to use for identifying the process group name in the metric. By default, it uses the base path of the executable.</p><p>The following template variables are available:<ul><li> {{.Comm}}:      Basename of the original executable from /proc/\<pid\>/stat.<li> {{.ExeBase}}:   Basename of the executable from argv[0].<li> {{.ExeFull}}:   Fully qualified path of the executable.<li> {{.Username}}:  Username of the effective user.<li> {{.Matches}}:   Map containing all regex capture groups resulting from matching a process with the cmdline rule group.<li> {{.PID}}:       PID of the process. Note that the PID is copied from the first executable found.<li> {{.StartTime}}: The start time of the process. This is useful when combined with PID as PIDS get reused over time.</ul></p></pre> | `{{.ExeBase}}` | no
```

Comment:
maybe do a short description in the table for long ones and more details below in paragraphs. I think that's how others are done in this situation.

Reply:
+1 try to keep the description in the table as short as possible so the table remains easy to view. 

Longer explanations can be done outside of the table.